deafrica/tests/test_chirps.py

```diff
@@ -21,16 +21,26 @@ def test_one_full(remote_file):
 
     year = "2018"
     month = "09"
+    day = "01"
 
-    s3_dst = f"s3://{TEST_BUCKET_NAME}"
+    s3_monthly_dst = f"s3://{TEST_BUCKET_NAME}"
+    s3_daily_dst = f"s3://{TEST_BUCKET_NAME}"
 
-    download_and_cog_chirps(year, month, s3_dst, overwrite=True)
+    download_and_cog_chirps(year, month, s3_monthly_dst, s3_daily_dst, overwrite=True, daily=True)
```

Comment:
I think this test should be split into two, one that does daily and one that does monthly.

Can we limit the daily one to only do one day?